05-html-css/vision-accessibility.md

```diff
@@ -0,0 +1,11 @@
+# Vision Accessibility & Using Screen Readers
```

Comment:
Would it possibly also be relevant to demonstrate a contrast ratio tool? http://leaverou.github.io/contrast-ratio/